webkom/lego-webapp
# Update company design

# Description

Updates the company details page in BDB to use the new table components, and applies a more modern and compact design. Feel free to suggest design changes if you have any! 😄 

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

## Before:
[Screencast from 2024-09-22 19-41-43.webm](https://github.com/user-attachments/assets/edb180b5-e91a-402d-940a-31bec30c8259)

## After:
[Screencast from 2024-09-22 19-38-55.webm](https://github.com/user-attachments/assets/ce8151af-b00e-46cb-adbf-b063146911e1)

# Testing


1. Navigate to a company detail page.
2. Verify that all table functionality works.
3. Verify that all information is correctly displayed.

Resolves [ABA-1054](https://linear.app/abakus-webkom/issue/ABA-1054/improve-single-company-page)
